ocaml/xapi/test_vlan.ml

```diff
@@ -170,10 +190,10 @@ let test =
     "test_create_internal" >:: test_create_internal;
     "test_create_unmanged_pif" >:: test_create_unmanaged_pif;
     "test_create_network_already_connected" >:: test_create_network_already_connected;
-    "test_create_pif_not_a_bond_slave" >:: test_create_pif_not_a_bond_slave;
-    "test_create_pif_not_vlan_slave" >:: test_create_pif_not_vlan_slave;
+    "test_create_pif_is_bond_slave" >:: test_create_pif_is_bond_slave;
```

Comment:
Why did you rename these tests names?
Shouldn't we add the new tests in addition to the old ones instead?

Reply:
I am renaming the test with correct name, for example, `test_create_pif_not_a_bond_slave` the original writer aims to test the scenario: cannot create vlan on a bond slave PIF. The test case actually test creating vlan on a PIF which is a bond slave, so I rename it to `test_create_pif_is_bond_slave`